File: specification/web/resource-manager/Microsoft.Web/stable/2022-03-01/examples-js/AppServiceEnvironments_GetAseV3NetworkingConfiguration.js
const { WebSiteManagementClient } = require("@azure/arm-appservice");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Description for Get networking configuration of an App Service Environment
 *
 * @summary Description for Get networking configuration of an App Service Environment
 * x-ms-original-file: specification/web/resource-manager/Microsoft.Web/stable/2022-03-01/examples/AppServiceEnvironments_GetAseV3NetworkingConfiguration.json
 */
async function getNetworkingConfigurationOfAnAppServiceEnvironment() {
  const subscriptionId = "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345";
  const resourceGroupName = "test-rg";
  const name = "test-ase";
  const credential = new DefaultAzureCredential();
  const client = new WebSiteManagementClient(credential, subscriptionId);
  const result = await client.appServiceEnvironments.getAseV3NetworkingConfiguration(
    resourceGroupName,
    name
  );
  console.log(result);
}

getNetworkingConfigurationOfAnAppServiceEnvironment().catch(console.error);

File: specification/web/resource-manager/Microsoft.Web/stable/2022-03-01/examples-java/AppServiceEnvironments_GetAseV3NetworkingConfiguration.java
import com.azure.core.util.Context;

/** Samples for AppServiceEnvironments GetAseV3NetworkingConfiguration. */
public final class Main {
    /*
     * x-ms-original-file: specification/web/resource-manager/Microsoft.Web/stable/2022-03-01/examples/AppServiceEnvironments_GetAseV3NetworkingConfiguration.json
     */
    /**
     * Sample code: Get networking configuration of an App Service Environment.
     *
     * @param azure The entry point for accessing resource management APIs in Azure.
     */
    public static void getNetworkingConfigurationOfAnAppServiceEnvironment(
        com.azure.resourcemanager.AzureResourceManager azure) {
        azure
            .webApps()
            .manager()
            .serviceClient()
            .getAppServiceEnvironments()
            .getAseV3NetworkingConfigurationWithResponse("test-rg", "test-ase", Context.NONE);
    }
}
